<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777201</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans KR</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>운동도와조</string>
  </property>
  <property name="windowOpacity">
   <double>4.000000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>31</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:'#3c4043';</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>70</y>
     <width>130</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:'black'; background-color:&quot;white&quot;;
border-top-left-radius: 9px;
border-bottom-left-radius: 9px;</string>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string> Level</string>
   </property>
   <property name="indent">
    <number>-13</number>
   </property>
  </widget>
  <widget class="QLabel" name="level">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>70</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:'black'; background-color:&quot;white&quot;;</string>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string> :  1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>104</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:'black'; background-color:&quot;white&quot;;
border-top-right-radius:9px;
border-bottom-right-radius:9px;</string>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string> 통과</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>310</y>
     <width>130</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:'black'; background-color:&quot;white&quot;;
border-top-left-radius: 9px;
border-bottom-left-radius: 9px;</string>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string> 다음운동</string>
   </property>
  </widget>
  <widget class="QLabel" name="second">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:'red';</string>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string>&quot;&quot;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>190</y>
     <width>130</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:'black'; background-color:&quot;white&quot;;
border-top-left-radius: 9px;
border-bottom-left-radius: 9px;</string>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string> 평균 정확도 </string>
   </property>
  </widget>
  <widget class="QLabel" name="avg_score">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>190</y>
     <width>191</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:'black'; background-color:&quot;white&quot;;
border-top-right-radius:9px;
border-bottom-right-radius:9px;</string>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string> :  90</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:'white';</string>
   </property>
   <property name="midLineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string> 초 뒤에 진행합니다. </string>
   </property>
  </widget>
  <widget class="QLabel" name="next_ex">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>310</y>
     <width>191</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:'black'; background-color:&quot;white&quot;;
border-top-right-radius:9px;
border-bottom-right-radius:9px;</string>
   </property>
   <property name="text">
    <string> :  운동 이름 </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
